packages/assets-controllers/CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Add token detection support
   - Add NFT detection support
 
+### Changed
+
+- **UPDATE:** Refactor `TokenRatesController` to support processing multiple chains simultaneously. The controller now accepts an array of chain IDs and tickers instead of a single value, streamlining the polling process by iterating over all chains in one loop ([#5645](https://github.com/MetaMask/core/pull/5645))
```

Comment:
Is this update referring to the change of executePoll here https://github.com/MetaMask/core/pull/5645/files#diff-80acb3577438ad64c78c3ff6d7d622d18b0b1136b0e2e7102852a8c1b342cf99R645 ?

Reply:
Alright the Update entry in the changelog seemed to me like its duplicate of the Breaking entry;
I think they both refer to the changes to the polling loop?
If that is correct maybe we keep just one? the Breaking entry?